docs/11-decorators.md

```diff
@@ -53,6 +53,48 @@ class PostDecorator
 end
 ```
 
+If you include comments on resources, your decorator class must respond to
+`model` returning a resource, and `decorated?` returning true.
+
+```ruby
+# app/decorators/post_decorator.rb
+class PostDecorator
+  attr_reader :post
+  delegate_missing_to :post
+
+  def initialize(post)
+    @post = post
+  end
+
+  def decorated?
+    true
+  end
+
+  def model
+    post
+  end
+end
+```
+
+If you use any actions with param(e.g. show, edit, destroy), your decorator
+class must respond to `to_param`.
```

Comment:
Are you saying that for any action with a default id parameter, the decorator requires this? Or its only required if you choose a different parameter other then the default id?

Reply:
I intended the former.

Reply:
For example, if you choose a different parameter as follows,

```ruby
ActiveAdmin.register Post do
  decorate_with PostDecorator
  
  controller do
    def find_resource
      scoped_collection.where(name: params[:id]).first!
    end
  end
end

# app/models/post.rb
class Post < ApplicationRecord
  def to_param
    name
  end
end
```

your decorator class require `to_param` and returning name.

```ruby
class PostDecorator
  attr_reader :post
  delegate_missing_to :post

  def initialize(post)
    @post = post
  end

  def to_param
    post.name
  end
end
```

Reply:
Sorry, I'm still confused by this one since you said its the former but showing where the parameter is customized. Are you saying that if `to_param` is overridden in the model, then the decorator should also respond to that method? Or actually just delegate to it no?

Reply:
Sorry for the lack of clarity 🙏 
PORO decorator always includes Object class in ancestor classes, so it can respond to `to_param`.
This means that if you expect the PORO decorator's `to_param` to act as decorated model's `to_param`, we need to explicitly delegate `to_param` to the decorated model.
From the above, I realized that my proposed code could be a bit more concise and clear.

```diff
  # app/decorators/post_decorator.rb
  class PostDecorator
    attr_reader :post
    delegate_missing_to :post
+   delegate :to_param, to: :post

    def initialize(post)
      @post = post
    end
- 
-   def to_param
-     post.id
-   end
  end
```

Reply:
The additional clarification makes sense to me, can you include it in this patch?

Reply:
Thank you for your feedback, I update this path!

cf. https://github.com/activeadmin/activeadmin/pull/7556/commits/d7c8a38acd563fc4e14aa9307f7b371d6d7febe9